backend/main.py

```
@@ -71,151 +137,332 @@ def get_spots(
-    # メモリ DB から取得したスポットを軽量化して返す
-    lightweight_spots: List[schemas.SpotResponse] = []
-    for spot in fake_spots_db:
-        spot_lite = spot.model_copy(
-            update={"content": spot.content.model_copy(update={"description": None})}
-        )
-        lightweight_spots.append(spot_lite)
 
-    return lightweight_spots
+@app.post("/upload/image")
+async def upload_image(
+    file: UploadFile = File(...),
```

Ruff (0.14.7)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)